src/main/scala/mesosphere/marathon/core/event/http/impl/ActorHttpCallbackSubscriptionService.scala

```diff
@@ -0,0 +1,30 @@
+package mesosphere.marathon.core.event.http.impl
+
+import akka.actor.ActorRef
+import akka.event.EventStream
+import akka.pattern.ask
+import mesosphere.marathon.core.event.http.{ EventSubscribers, HttpCallbackSubscriptionService }
+import mesosphere.marathon.core.event.http.impl.SubscribersKeeperActor.GetSubscribers
+import mesosphere.marathon.core.event.{ EventConf, _ }
+
+import scala.concurrent.Future
+
+class ActorHttpCallbackSubscriptionService(
```

Comment:
To be consistent, this class should be called: `HttpCallbackSubscriptionServiceDelegate`